ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -37,6 +37,44 @@ readonlyrest:
+
+    # Requirement A (RW access to "tenant1" tenant Kibana indices)
+    - name: kib_tracy_tenant1_adm_ext98107
+      groups_or: ["kib_tracy_tenant1_adm_ext98107"]
+      kibana: 
+        access: rw
+        index: ".kibana_tracy_tenant1"
+
+    - name: els_tracy_adm_ext98107_allow
+      type: allow
+      groups_or: ["els_tracy_adm_ext98107"]
+      indices: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)